packages/config/test/options.test.js

```diff
@@ -256,6 +256,18 @@ describe('config: options', () => {
   })
 })
 
+describe('config: serverMiddleware', () => {
+  test('should transform serverMiddleware hash', () => {
+    const serverMiddleware = {
+      '/resource' (req, res, next) {
```

Comment:
Although this is fine, I'd prefer to just use a normal key/value syntax with a closure. Putting single quotes around the function name is just weird

Reply:
@pimlie heh, noted -- mostly personal preference, either way works, but I'll update it for you.

Reply:
You could also just always prepend the first slash, so users dont have to bother with that. Then you could do the above by just using `resource`.

Just have to change the [replace regex here](https://github.com/nuxt/nuxt.js/blob/1b837a1ab41342016800e18c7ac7c05ad11afb9d/packages/server/src/server.js#L201-L204) to read `replace(/\/{2,}/g, '/')`

Reply:
I'm opting to keep it literal unless anyone feels strongly otherwise -- I think the syntax can grow on you if you give it time, and it saves us from the confusion.